image.go

```diff
@@ -36,6 +36,7 @@ type Image struct {
 	Attachment string     `json:"attachment,omitempty"`
 	Filename   string     `json:"filename,omitempty"`
 	VariantIds []int64    `json:"variant_ids,omitempty"`
+	AdminGraphqlApiId     `json:"admin_graphql_api_id,omitempty"`
```

Comment:
Ah yes, this should be `AdminGraphqlApiId string  ....`

Reply:
Yeah my bad, I was editing the file directly on GitHub 🥲